src/main/java/com/webdev/cosmo/cosmobackend/security/FacebookAuthentication.java

```diff
@@ -0,0 +1,58 @@
+package com.webdev.cosmo.cosmobackend.security;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+
+import java.util.Collection;
+
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class FacebookAuthentication implements Authentication {
+
+    private boolean authenticated = false;
```

Comment:
I think it is better to pass authenticated as object in constructor anyway and depend on spring unauthorized in such case